packages/emptystate/src/svg/document-small.svg

```diff
@@ -1,9 +1,9 @@
-    <path fill="#FFF" d="M20,9.03127 C20,9.01924 20.00052,9.00895 20.00129,9.00028 C20.00548,9.0001 20.01005,9 20.015,9 L50.1605,9 C50.1655,9 50.17,9.0001 50.1742,9.00028 C50.175,9.00895 50.1755,9.01924 50.1755,9.03127 L50.1755,49.6632 C50.1755,49.6719 50.1747,49.6768 50.1742,49.6788 L50.1737,49.6809 L50.1735,49.6814 L50.1735,49.6815 L50.1731,49.682 L50.1723,49.6831 L50.1702,49.6852 C50.1666,49.6887 50.1616,49.6919 50.1567,49.6939 L50.1553,49.6944 L47.1458,49.6944 L47.1458,51.6944 L50.1605,51.6944 C51.1696,51.6944 52.1755,50.8836 52.1755,49.6632 L52.1755,9.03127 C52.1755,8.49937 51.9971,7.976327 51.6091,7.582135 C51.2197,7.186525 50.6971,7 50.1605,7 L20.015,7 C19.47843,7 18.95579,7.186525 18.56642,7.582135 C18.17845,7.976327 18,8.49937 18,9.03127 L18,13 L20,13 L20,9.03127 Z"/>
-    <path stroke="#FFF" stroke-linecap="round" stroke-width="2" d="M30 23L40 23M19 28L40 28M19 33L40 33M19 38L40 38M19 43L40 43M19 48L31 48"/>
-    <rect width="7" height="4" x="18" y="19" fill="#FFF" fill-opacity=".2" stroke="#FFF" stroke-width="2" rx="1"/>
-    <polygon fill="#C4C4C4" points="13 56.601 13 49.175 45.175 17 45.175 28.756 45.175 56.601" opacity=".2"/>
+    <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M44.6104,56.6277 L14.02965,56.6277 C13.41186,56.6277 13,56.2158 13,55.598 L13,15.02965 C13,14.41186 13.41186,14 14.02965,14 L44.6104,14 C45.2282,14 45.64,14.41186 45.64,15.02965 L45.64,55.598 C45.64,56.2158 45.1252,56.6277 44.6104,56.6277 Z"/>
+    <path fill="currentColor" d="M20,9.03127 C20,9.01924 20.00052,9.00895 20.00129,9.00028 C20.00548,9.0001 20.01005,9 20.015,9 L50.1605,9 C50.1655,9 50.17,9.0001 50.1742,9.00028 C50.175,9.00895 50.1755,9.01924 50.1755,9.03127 L50.1755,49.6632 C50.1755,49.6719 50.1747,49.6768 50.1742,49.6788 L50.1737,49.6809 L50.1735,49.6814 L50.1735,49.6815 L50.1731,49.682 L50.1723,49.6831 L50.1702,49.6852 C50.1666,49.6887 50.1616,49.6919 50.1567,49.6939 L50.1553,49.6944 L47.1458,49.6944 L47.1458,51.6944 L50.1605,51.6944 C51.1696,51.6944 52.1755,50.8836 52.1755,49.6632 L52.1755,9.03127 C52.1755,8.49937 51.9971,7.976327 51.6091,7.582135 C51.2197,7.186525 50.6971,7 50.1605,7 L20.015,7 C19.47843,7 18.95579,7.186525 18.56642,7.582135 C18.17845,7.976327 18,8.49937 18,9.03127 L18,13 L20,13 L20,9.03127 Z"/>
+    <path stroke="currentColor" stroke-linecap="round" stroke-width="2" d="M30 23L40 23M19 28L40 28M19 33L40 33M19 38L40 38M19 43L40 43M19 48L31 48"/>
+    <rect width="7" height="4" x="18" y="19" fill="currentColor" fill-opacity=".2" stroke="currentColor" stroke-width="2" rx="1"/>
+    <polygon fill="currentColor" points="13 56.601 13 49.175 45.175 17 45.175 28.756 45.175 56.601" opacity=".1"/>
```

Comment:
Not sure you meant to change this one? See svgr config https://github.com/pluralsight/design-system/blob/dc50ea0178ed2e8dca0aa82949c568ed5786380c/packages/emptystate/svgr.config.js#L33-L36

Reply:
I did mean too. looks like i missed it the first time around. I also updated the opacity to get the right shade. thanks for double checking